### PR TITLE
[DB-13163] Added pg_stat_statements extension installed check for db-queries-summary.psql script

### PR DESCRIPTION
**Pull request by sanyamsinghal**
- If installed, then execute the script. Else, assess-migration command will not generate any output for the UnsupportedQueryConstructs section

https://yugabyte.atlassian.net/browse/DB-13163

### CODE REVIEW

File: yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql [hunk 1]
```diff
@@ -15,6 +13,6 @@ FROM
 WHERE
     dbid = (SELECT oid FROM pg_database WHERE datname = current_database());
 
-\copy temp_table to 'db_queries_summary.csv' WITH CSV HEADER;
+\copy temp_table to 'db-queries-summary.csv' WITH CSV HEADER;
```
Comment (makalaaneesh) [inline yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: Won't this also require a change in go code where you're reading the CSV?

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: we are just looking for files matching pattern - `*.csv`

```
func populateMetadataCSVIntoAssessmentDB() error {
	metadataFilesPath, err := filepath.Glob(filepath.Join(assessmentMetadataDir, "*.csv"))
	if err != nil {
		return fmt.Errorf("error looking for csv files in directory %s: %w", assessmentMetadataDir, err)
	}
```

Comment (makalaaneesh) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: Hmm, but I see this in our code? 
`DB_QUERIES_SUMMARY       = "db_queries_summary"` ?

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: yeah that variable is for table names in sqlite.
Tables cant have `-` in name

```
const (
	TABLE_INDEX_IOPS         = "table_index_iops"
	TABLE_INDEX_SIZES        = "table_index_sizes"
	TABLE_ROW_COUNTS         = "table_row_counts"
	TABLE_COLUMNS_COUNT      = "table_columns_count"
	INDEX_TO_TABLE_MAPPING   = "index_to_table_mapping"
	OBJECT_TYPE_MAPPING      = "object_type_mapping"
	TABLE_COLUMNS_DATA_TYPES = "table_columns_data_types"
	TABLE_INDEX_STATS        = "table_index_stats"
	DB_QUERIES_SUMMARY       = "db_queries_summary"

	PARTITIONED_TABLE_OBJECT_TYPE = "partitioned table"
	PARTITIONED_INDEX_OBJECT_TYPE = "partitioned index"
)
```

Comment (makalaaneesh) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: So, the table name is automatically generated by replacing - with _?

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: not automatically. Voyager code do CREATE TABLE in SQLITE assessmentdb there we define it with `_`

Comment (sanyamsinghal) [reply yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/db-queries-summary.psql:16]: oh right, that we do while mapping csv name to table name
```
func populateMetadataCSVIntoAssessmentDB() error {
	metadataFilesPath, err := filepath.Glob(filepath.Join(assessmentMetadataDir, "*.csv"))
	if err != nil {
		return fmt.Errorf("error looking for csv files in directory %s: %w", assessmentMetadataDir, err)
	}

	for _, metadataFilePath := range metadataFilesPath {
		baseFileName := filepath.Base(metadataFilePath)
		metric := strings.TrimSuffix(baseFileName, filepath.Ext(baseFileName))
		tableName := strings.Replace(metric, "-", "_", -1)
		// collecting both initial and final measurement in the same table
		tableName = lo.Ternary(strings.Contains(tableName, migassessment.TABLE_INDEX_IOPS),
			migassessment.TABLE_INDEX_IOPS, tableName)

		log.Infof("populating metadata from file %s into table %s", metadataFilePath, tableName)
		file, err := os.Open(metadataFilePath)
		if err != nil {
			log.Warnf("error opening file %s: %v", metadataFilePath, err)
			return nil
		}
		csvReader := csv.NewReader(file)
		csvReader.ReuseRecord = true
		rows, err := csvReader.ReadAll()
		if err != nil {
			log.Errorf("error reading csv file %s: %v", metadataFilePath, err)
			return fmt.Errorf("error reading csv file %s: %w", metadataFilePath, err)
		}

		err = assessmentDB.BulkInsert(tableName, rows)
		if err != nil {
			return fmt.Errorf("error bulk inserting data into %s table: %w", tableName, err)
		}
		log.Infof("populated metadata from file %s into table %s", metadataFilePath, tableName)
	}

	err = assessmentDB.PopulateMigrationAssessmentStats()
	if err != nil {
		return fmt.Errorf("failed to populate migration assessment stats: %w", err)
	}
	return nil
}
```

**Review by makalaaneesh — APPROVED** [review]

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3775/

**Pull request merged**